### ISSUE TITLE
Attempts don't reset even after maxInterval is reached

### ISSUE THREAD

**Issue by Filliposniper** [saintedlama/passport-local-mongoose#265]
I am having issues with my logins, am trying to penalize a user from logging into their account on too many login attempts that's after a maxAttempts is reached, I noticed that if a user gives an incorrect login at first it logs out the normal incorrect error but on second attempt and details are correct the user is authenticated and attempts resets to 0 (perfect). BUT when a user exceeds the maxAttempts, the account is locked and attempts field never reset to 0 even after maxInterval is reached, and this locks out a user forever (:

```
var UserSchema = new mongoose.Schema({
     username:  "somebody"
     password: {type: String, minlength: 6},
})

UserSchema.plugin(passortLocalMongoose,{
    maxAttempts: 5,
    limitAttempts: true,
    maxInterval: 200000
});
```

Is there any field am missing out ?, or any code I need to write in order to implement this. Besides this every other thing works well. :)

**Comment by 1valdis** [comment]
I tried to do something similar as well and stuck to the same question. Seems like we should write unlock logic ourselves.

**Comment by Filliposniper** [comment]
@1valdis Yeah, maybe we have to write the logic ourselves, but it would have been better if it happens authomatically, just like when a user logs in :)

**Comment by omeraha** [comment]
Having the same issue. Any updates about it?

**Comment by blueyestar** [comment]
Save issue here.

**Comment by himanshu27budhiraja** [comment]
Facing the same issue in 2022!
